Cargo.toml

```diff
@@ -30,6 +30,8 @@ unicode-width = "0.1"
 ignore="0.4"
 crossbeam-channel = "0.4"
 walkdir="2.3"
+# todo use num_format instead
```

Comment:
https://crates.io/crates/num-format 

numformat  has way more downloads than thousands and is more configurable, we should probably use that.

Reply:
Yeah that'd probably be fine.  I actually was going to use that initially but figured I would go with a simpler crate.  They basically do the same thing for our usecase

Reply:
Frustatingly it looks like num-format doesn't build on a default windows Travis-ci box due to a weird clang dependency. 

I wanted num_format so 10,000 would be written as 10.000 in countries that use '.' instead of ',' but if it makes the CI this much bother I'll probably go back to the thousands crate.